test/test-project.js

```diff
@@ -53,9 +53,39 @@ test('package info gathering', function(t) {
   self.gather(function(err, project) {
     t.ifErr(err, 'should not error out');
     t.same(self, project);
+    t.equal(project.get('license'), 'MIT');
```

Comment:
I think in this case the small value gained in hygiene would be less than the additional cost of losing information. In this case, using literals gives us extra information in the places it is used such as type and value without having to find the declaration/initialization.